api/integrations/launch_darkly/client.py

```diff
@@ -35,7 +35,7 @@ def _iter_paginated_items(
         self,
         collection_endpoint: str,
         additional_params: Optional[dict[str, str]] = None,
-    ) -> Iterator[dict[str, Any]]:
+    ) -> Iterator[dict[str, Any] | str]:
```

Comment:
Fixed this for `getTags` endpoint.

Line 48:

```python
yield from response_json.get("items") or []
```

Here we assume `items` is another struct but for get tags, items is a list of strings.

Reply:
The correct typing fix for this would be something like:

```
T = TypeVar("T")

...

    def _iter_paginated_items(
        self,
        collection_endpoint: str,
        item_type: Type[T],
        additional_params: Optional[dict[str, str]] = None,
    ) -> Iterator[T]:

...


    # calling code
    list_of_environments = self._iter_paginated_items(collection_endpoint=endpoint, item_type=ld_types.Environment)
    list_of_str = self._iter_paginated_items(collection_endpoint=endpoint, item_type=str)
```